### PR TITLE
docs: mention workaround for bash_profile

### PR DESCRIPTION
**Pull request by antongolub**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -534,6 +534,18 @@ await $`pwd`
...
 ```
 
+#### Attaching .bash_profile/.zshrc
+
+By default `child_process` does not include aliases and bash functions (https://github.com/nodejs/node/issues/3916). 
+But you are still able to do it by hand. Just attach necessary directives to `$.prefix`.
+
+```js
+{
+  $.prefix = 'export NVM_DIR=$HOME/.nvm; source $NVM_DIR/nvm.sh; set -e;'
```
Comment (antonmedv) [inline README.md:544]: People will drop pipefall. Maybe better to suggest $.prefix += ...

Comment (antongolub) [reply README.md:544]: Reasonable.

File: README.md [hunk 2]
```diff
@@ -534,6 +534,18 @@ await $`pwd`
 EOF
 ```
 
+#### Attaching .bash_profile/.zshrc
+
+By default `child_process` does not include aliases and bash functions (https://github.com/nodejs/node/issues/3916).
```
Comment (antonmedv) [inline README.md:539]: I think link here unnecessarily as this behavior is actually expected. Not an "issue".

Comment (antongolub) [reply README.md]: ok

**Pull request merged**